certification/pyxis/submit.go

```diff
@@ -51,31 +51,30 @@ func (p *pyxisClient) SubmitResults(ctx context.Context, certInput *certificatio
 
 	// Create the image, or get it if it already exists.
 	certImage, err = p.createImage(ctx, certImage)
-	if err != nil && err != errors.ErrPyxis409StatusCode {
-		log.Error(err, "could not create image")
-		return nil, err
-	}
-
 	if err != nil && err == errors.ErrPyxis409StatusCode {
 		certImage, err = p.getImage(ctx, originalImageDigest)
 		if err != nil {
-			log.Error(err, "could not get image")
+			log.Error(fmt.Errorf("%w: could not get image", err))
 			return nil, err
 		}
 	}
+	if err != nil && err != errors.ErrPyxis409StatusCode {
+		log.Error(fmt.Errorf("%w: could not create image", err))
+		return nil, err
+	}
 
 	// Create the RPM manifest, or get it if it already exists.
 	rpmManifest := certInput.RpmManifest
 	rpmManifest.ImageID = certImage.ID
 	_, err = p.createRPMManifest(ctx, rpmManifest)
 	if err != nil && err != errors.ErrPyxis409StatusCode {
-		log.Error(err, "could not create rpm manifest")
+		log.Error(fmt.Errorf("%w: could not create rpm manifest", err))
 		return nil, err
 	}
 	if err != nil && err == errors.ErrPyxis409StatusCode {
 		_, err = p.getRPMManifest(ctx, rpmManifest.ImageID)
 		if err != nil {
-			log.Error(err, "could not get rpm manifest")
+			log.Error(fmt.Errorf("%w: could not get rpm manifest", err))
 			return nil, err
 		}
 	}
```

Comment:
This might also benefit from the previous comment on similar code, though I know this didn't really change here.